src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -73,6 +73,8 @@ public static class Features
             public const string CodeActionsSpellcheck = "CodeActions.Spellcheck";
             public const string CodeActionsSuppression = "CodeActions.Suppression";
             public const string CodeActionsUseAutoProperty = "CodeActions.UseAutoProperty";
+            public const string CodeActionsUseImplicitTyping = "CodeActions.UseImplicitTyping";
```

Comment:
We have two separate code actions?  That seems odd to me.  Why not one code action that simply applies the appropriate change?